src/system-script.js

```diff
@@ -1,22 +1,6 @@
-let nameCounter = 0;
-let scriptNameMap = {};
+let scriptNameMap = window.__systemAmdScript.scriptNameMap;
```

Comment:
Get the `scriptNameMap` that is defined in the other file. It is just a shared object in memory.

Reply:
Both sides can mutate it and read it.